lib/src/page_list_viewport_gestures.dart

```diff
@@ -198,10 +205,14 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
     // Choose to lock in a particular axis direction, or not.
     final movementAngle = translation.direction;
     final movementAnglePositive = movementAngle.abs();
-    if (((2 / 6) * pi < movementAnglePositive) && (movementAnglePositive < (4 / 6) * pi)) {
+    // only consider axis locking if the translation distance is sufficiently large
+    // this is FPS dependent, so 2 should be dewpendent on fps
```

Comment:
"dewpendent" -> "dependent", "fps" -> "FPS"

Please write comments like English sentences, following standard grammar rules, e.g., start with a capital letter.

Also, I can't tell what "So 2 should be dependent on FPS" means. Are you saying there's still work to do and this is a half-measure? Or are you stating something about the behavior of the code? If it's about code behavior, it's generally not a good idea to say "should" because that implies something might not happen. Instead, state with confidence what the code does.

Reply:
ok